packages/env-spec-parser/test/functions.test.ts

```diff
@@ -58,10 +58,21 @@ describe('function calls', functionValueTests({
       ITEM: 'foo-val',
     },
   },
+  'replace()': {
```

Comment:
This file is testing how the parser itself handles parsing functions, and we don't need to define or test the new resolver within the parser. The tests we want to add are in [env-graph](https://github.com/dmno-dev/varlock/blob/main/packages/varlock/env-graph/test/resolvers.test.ts)